modules/462-loki/docs/README.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "The loki module"
+---
+
+[Loki](https://grafana.com/oss/loki/) - module for storing and viewing logs.
```

Comment:
```suggestion
[Loki](https://grafana.com/oss/loki/) - module for storing and viewing logs.
```
Does the module allow you to view logs?  Maybe  - "The module for aggregating logs"?